source/css/scss/vendor_override/foundation/_buttons.scss

```diff
@@ -86,17 +85,21 @@ button, a.button, .button {
 }
 
 button.secondary, .button.secondary {
-	border-color: $kiva-text-dark;
+	border-color: $charcoal;
 	border-width: 1px;
 	font-size: $secondary-button-font-sm;
+  	padding: rem-calc(16) rem-calc(32);
+  	box-shadow: 0 1px #333;
```

Comment:
should we define a variable for this color and use it here? (same comment for other usage above)

Reply:
Good question that I'll bring up with @brandonatkiva. I hard code this value to #333 because with our last sweep of color updates we redefined $charcoal from #333 -> #484848. I thought it was weird we wanted to use a color we just removed from a variable, checked with Brandon who then verified that we wanted to use #333. So I can create a variable for that if needed, I'll just have to follow up again on why we're using a color we just removed from the defined palette.

Reply:
I've created a new variable in _palette.scss called $dark-charcoal: #333; and am using it in both places I previously hardcoded #333.